### DISCUSSION TITLE
Custom bot using the source code

### DISCUSSION THREAD

**Discussion by poke99catch** [Hyperclaw79/PokeBall-SelfBot#220] [Ideas]
i want to create a bot with this code hoe to do pls tell
:)

**Comment by Hyperclaw79** [comment]
You mean, an actual discord bot?
You can get the latest code after purchasing the Premium Version and I can support you on porting it into a Discord bot provided, proper accreditation is provided to this Github repo.

**Reply by poke99catch** [reply]
send me discord link and is their no free version umm
how much is Premium Version

**Reply by Hyperclaw79** [reply]
All the details are available on the Readme under the [Donations](https://github.com/Hyperclaw79/PokeBall-SelfBot-Poketwo-Automation#Donations) section
